spec/integration/manifest.spec.yml

```diff
@@ -174,6 +174,38 @@ spec:
             enabled:
               type: boolean
               default: false
+            organization:
+              description: >
+                The responsible organization of the integration. This is used to tag the agentless agent deployments
+                for monitoring.
+              type: string
+              examples:
+                - "security"
+            division:
+              description: >
+                The division responsible for the integration. This is used to tag the agentless agent deployments
+                for monitoring.
+              type: string
+              examples:
+                - "cloud-security"
+            team:
+              description: >
+                The team responsible for the integration. This is used to tag the agentless
+                agent deployments for monitoring.
+              type: string
+              examples:
+                - "cloud-security-posture-management"
+            allOf:
+              - if:
+                  properties:
+                    enabled:
+                      const: true
+                then:
+                  required:
+                    - organization
+                    - division
+                    - team
```

Comment:
Setting these new fields as required may break existing packages using `agentless`, do we have any?

Reply:
Not at them moment.  AFAIK there are some integrations like Okta and Cloud Connector that want to use this setting.

@eyalkraft can you confirm?

Reply:
> How could we still make these fields required? Can the integrations be updated first? Or could we provide a default value?

We cannot update the integrations first because the field is not allowed yet. We have two options:
* We add the fields as now, but we make them mandatory only for packages using spec >= 3.2.3. Then integrations can add the fields, but they are only required on newer versions of the spec. This would be the less "breaking" approach, but we can have packages with older versions of the spec and without these fields, what would need to be handled by agentless code.
* We make the breaking change, so these fields are required. Then when updating the package-spec in the integrations repository, we need to add the fields where they are missing. This option is more "breaking", but probably better in the long term, but we need to know what fields to use when updating the package-spec.

Regarding default fields, we can add a default to the spec as documentation, but its implementation depends on whatever reads these settings, defaults are not enforced by the spec.

Reply:
@jsoriano, I had an offline conversation with @eyalkraft, we decided to change all 3 integrations that have or will have the `depoyment_mode` attribute.